Updates to the Viewport Interaction Model documentation (#990)
o3de/o3de.org: content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md

You can use the Viewport Interaction Model to create reference spaces for your selected entities. This enables you to customize your selected entities in relation to a reference space.

When working with reference spaces, remember the following rules:
+ New selections always default to local space.
+ Holding **Shift** aligns the manipulator to the world coordinate space.
+ Holding **Alt** changes the manipulator to have _individual_ influence as opposed to _group_ influnce (the default).

> + You can easily access custom reference spaces by moving the manipulator (press and hold **Ctrl** and click and drag the manipulaotr) or picking a target entity as a reference space (press hold **Ctrl** and **Alt** and click a target entity).

Reviewer: ```suggestion
+ You can access custom reference spaces by moving the manipulator (press and hold **Ctrl** and then choose and drag the manipulator) or picking a target entity as a reference space (press hold **Ctrl** and **Alt** and click a target entity).
```

> The Viewport Interaction Model simplifies the mental model for you and doesn't require you to keep track manually of the last space (local or world) that you were using. You can specify a target and define that target as custom reference spaces. This new reference space model covers any arbitrary transformation.

Reviewer: ```suggestion
The Viewport Interaction Model simplifies the mental model for you by not requiring you to manually track the last local or world space that you were using. You can specify a target and define that target as custom reference spaces. This new reference space model covers any arbitrary transformation.
```

## Switching Between Local and World Space 

You can use the manipulator to switch between local and world space without losing focus of your entity selection in the viewport.

**To switch between local and world space**

1. In the viewport, select a child entity that has a parent.

1. Use the manipulator to modify the entity.

1. To switch to world space, press and hold **Shift**. Use the manipulator to modify the entity.

> 1. Viewport UI (W/P/L _World, Parent, Local_ in the top right of the viewport) allow you to pin a reference space.

Reviewer 2: We have nifty little icons we can insert now.

```suggestion
1. Viewport UI icons ({{< icon "world.svg" >}} World, {{< icon "parent.svg" >}} Parent, and {{< icon "local.svg" >}} Local) in the top right of the viewport allow you to pin a reference space.
```

Author: Okay nice, do we have these icons already? Or will I need to find and add them?

Author: I found the icon and checked and it is actually white, so might display poorly on the page (it'll just be invisible to the reader). Is there a solution for this?
